### PR TITLE
Feature/csv reader

### PR DESCRIPTION
**Pull request by Elian012**
Prints a content of a csv reader in a specific way 
-----| ---- | 
--    |

### CODE REVIEW

File: read_handler.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""
+ Author: Elian
+ Purpose: A csv file reader handler.
+ Date: 04.03.2024
+"""
+
+
+def csv_read_handler(cell: str, numb_of_character: int) -> None:
```
Comment (netanelezra) [inline read_handler.py]: CR: Choose better name for the second argument.

Comment (netanelezra) [inline read_handler.py]: CR: Choose more indicative name for the function.

File: read_handler.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
```
Comment (netanelezra) [inline read_handler.py]: CR: Is this really the best name for this module? 
Maybe we should put it in utils folder?

**Review by netanelezra — CHANGES_REQUESTED** [review]
Great work, Fix the comments

File: csv_file_read_handler.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+"""
```
Comment (netanelezra) [inline csv_file_read_handler.py]: Choose a better name for the module.

**Review by netanelezra — CHANGES_REQUESTED** [review]

File: csv_file_read_handler.py (hunk 3 above)
Comment (Elian012) [reply csv_file_read_handler.py]: What's considered a good or better name?

Comment (Elian012) [reply csv_file_read_handler.py]: Could print_csv be a better name?

Comment (netanelezra) [reply csv_file_read_handler.py]: Describe to me what is the purpose of the file?

Comment (Elian012) [reply csv_file_read_handler.py]: I created the file to help me print the csv file cells cause I needed to print them in a specific way, so it only contains the printing function.

Comment (netanelezra) [reply csv_file_read_handler.py]: so maybe include the word util in the name ?

Comment (Elian012) [reply csv_file_read_handler.py]: so woud csv_print_util be okay?

Comment (netanelezra) [reply csv_file_read_handler.py]: great

**Pull request merged**